travis_install.sh

```diff
@@ -7,3 +8,4 @@ elif [ $(python -c "import sys; print(sys.version_info.major)") = "3" ] && [ "$T
   python setup.py bdist_wheel --cgal-dir=/cgal --keep-temp
   delocate-wheel -w fixed_wheel dist/*.whl 
 fi
+set +x
```

Comment:
Nothing happens after that line: the shell quits. ... Unless the file `travis_install.sh` is sourced and not executed.